README.md

```diff
@@ -1,2 +1,18 @@
 # qlink
 A full rewrite of the original Qlink app that started it all.
+
+## Build & run
+Make sure you're on Flutter 2.0 before running `flutter run`.
+
+## Project structure
+```sh
+├── android/
+├── ios/
+├── web/
+├── lib/
+│   ├── screens/
+│   ├── theme.dart
+│   └── main.dart
```

Comment:
Don't know how heavy this app will get, but a workspace manager that prescribes a certain directory structure/organization for files (or even with file generation) could be nice.

Or just follow some guide like https://medium.com/flutter-community/flutter-scalable-folder-files-structure-8f860faafebd

Reply:
I was thinking we could add folders as necessary:
```
├── widgets/
├── services/
```

I will try to find a convention when I add `bloc` into the project.

I don't think this project will get heavy at all, so a "workspace manager" might be overkill.